docs/src/modules/java/pages/value-entity.adoc

```diff
@@ -41,8 +41,8 @@ Let us start by showing how to create the Value Entity:
 ----
 include::example$java-spring-valueentity-counter/src/main/java/com/example/CounterEntity.java[tags=declarations]
 ----
-<1> Every Entity must be annotated with `@EntityType` with a stable name. This name should be unique among the different existing entities within a Kalix application.
-<2> The `@EntityKey` value should be unique per entity and map to some field being received on the route path, in this example it's the `counter_id`.
+<1> Every Entity must be annotated with `@TypeId` with a stable identifier. This identifier should be unique among the different existing entities within a Kalix application.
```

Comment:
I would change the order

Reply:
But that's a lot of work. It's just a visual aspect.

Reply:
Yeah, I agree, but will leave it as is. Not work to go change the callouts.